package/network/services/dnsmasq/Makefile

```diff
@@ -110,6 +112,9 @@ define Package/dnsmasq-full/config
 	config PACKAGE_dnsmasq_full_ipset
 		bool "Build with IPset support."
 		default y
```

Comment:
Since `nftables` is the new default, it might be a good idea to disable IPset support by default.